droidlet/interpreter/robot/default_behaviors.py

```diff
@@ -36,18 +37,29 @@ def init_logger():
 
 init_logger()
 
+#TODO: Move these utils to a suitable place
+def add_or_replace(agent, pred_text, obj_text):
```

Comment:
I think it would be reasonable to put this as a class method of TripleNode

Reply:
updated the comment, will do this as a follow-up to limit the scope of this PR